packages/react-components/react-card/src/components/Card/useCard.ts

```diff
@@ -26,15 +26,16 @@ export const useCard_unstable = (props: CardProps, ref: React.Ref<HTMLElement>):
     tabBehavior: focusMap[focusMode],
   });
 
+  const focusAttrs = focusMode !== 'off' ? { tabIndex: 0, ...groupperAttrs } : {};
```

Comment:
**suggestion:**

Using empty javascript object as a value is dangerous, as this translates to former bottom type `{}` in typescript context. Now I lost all type information. This might be even bigger issues as `getNativeElementProps` is completely type unaware.

![image](https://user-images.githubusercontent.com/1223799/168590613-2e372a9e-7225-45b2-a0d5-648a36fdfd31.png)

Can we use `null`  to be explicit about empty-ness and improve type information as a side effect ?

![image](https://user-images.githubusercontent.com/1223799/168591213-5b1263df-3ed2-4425-9169-d867ea939d2c.png)